spring-xd-module/src/main/java/org/springframework/xd/module/options/DefaultModuleOptionsMetadataResolver.java

```diff
@@ -109,6 +109,7 @@
 	 */
 	public DefaultModuleOptionsMetadataResolver(ConversionService conversionService) {
 		Assert.notNull(conversionService, "The provided conversionService must not be null.");
+		this.conversionService = conversionService;
```

Comment:
@izeye thanks for the PR. I believe this constructor is not being used and can be removed. @ericbottard Could you confirm?